homepage/de.js

```diff
@@ -0,0 +1,67 @@
+	"Developer friendly": "Entwicklerfreundlich",
+	"This is a picture of a cloud against a background of clouds. OMG! We <a %s>integrate</a> with a bunch of stuff in the cloud.": "Das ist ein Bild einer Wolke vor einem Hintergrund aus Wolken. OMG! Wir <a %s>integrieren</a> eine Menge an Dingen aus der Cloud.",
+	"Some of our communities": "Einige unserer Communities",
+	"Create your own": "Eigenen Chatroom erstellen",
+	"Explore rooms": "Chatrooms entdecken",
+	"Explore": "Entdecken",
+	"Integrated with your workflow": "In deinen Arbeitsablauf eingebunden",
+	"Gitter is built on top of GitHub and is tightly integrated with your organisations, repositories, issues and activity.": "Gitter wurde auf Basis von GitHub entwickelt und ist eng mit deinen GitHub Organisations, Repositories, Issues und Aktivit&auml;ten verbunden.",
+	"We provide integrations with Trello, Jenkins, Travis CI, Heroku, Sentry, BitBucket, HuBoard, Logentries, Pagerduty &amp; Sprintly. We also accept custom webhooks, have an <a %s>open source repository</a> for integrations as well as a <a %s>flexible API</a>.": "Wir bieten Integrationen mit Trello, Jenkins, Travis CI, Heroku, Sentry, BitBucket, HuBoard, Logentries, Pagerduty und Sprintly an. Wir akzeptieren auch eigene webhooks, haben ein <a %s>Open Source Repository</a> f&uuml;r Integrationen genauso wie eine <a %s>flexible API</a>.",
+	"Loved by our users": "Von unseren Nutzern geliebt",
+	"Gitter is nothing without the people using it day in and day out.": "Gitter ist nichts ohne die Nutzer die es Tag f&uuml;r Tag verwenden.",
+	"We asked a few what Gitter means to them, this is what they had to say. If you love Gitter, Tweet us <a %s>%s</a> and we'll update these quotes every month.": "Wir fragten einige Entwickler, was Gitter f&uuml;r sie bedeutet.  Dies ist was sie zu Gitter sagen. Wenn du Gitter liebst, Tweete uns <a %s>%s</a>. Wir aktualisieren diese Meinungen jeden Monat.",
+	"Personal pricing": "Pers&ouml;nlicher Preis",
+	"Free": "Kostenlos",
+	"Forever": "F&uuml;r immer",
+	"<b>Unlimited</b> public rooms": "<b>Unbegrenzt</b> &ouml;ffentliche Chatrooms",
+	"<b>1</b> private room": "<b>1</b> privater Chatroom",
+	"<b>2 weeks’</b> chat history": "<b>2 w&ouml;chigee</b> Chat-Verlauf",
+	"<b>Get started</b> for free": "Kostenlos <b>registrieren</b>",
+	"Individual": "einzeln",
+	"monthly": "monatlich",
+	"<b>Unlimited</b> private rooms": "<b>Unbegrenzte</b> private Chatrooms",
+	"<b>Unlimited</b> chat history": "<b>Unbegrenzter</b> Chat-Verlauf",
+	"<b>Purchase</b> now": "Jetzt <b>kaufen</b>",
+	"Organisation pricing": "Preispl&auml;ne",
+	"Bronze": "Bronze",
+	"per user/month": "pro Benutzer/Monat",
+	"<b>6 months’</b> chat history": "<b>6 monatiger</b> Chat-Verlauf",
+	"Silver": "Silber",
+	"Our free plan includes one private room per organisation you belong to on GitHub": "Unser kostenloses Angebot beinhaltet einen kostenlosen Chatroom pro Organisation der du auf Github angeh&ouml;rst",
```

Comment:
„… einen kostenlosen *nichtöffentlichen* Chatroom …“, we get unlimited public rooms :)

and comma after „Organisation“?